<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWidget</class>
 <widget class="QWidget" name="PlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chessboard</string>
  </property>
  <widget class="QLabel" name="labelBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>116</green>
         <blue>108</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LabelForShowNoUse</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyButton" name="buttonGoBack" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyButton" name="buttonStop" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyButton" name="buttonExit" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyButton" name="buttonRegret" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyButton" name="buttonRedo" native="true">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>400</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberBlack">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="value" stdset="0">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberWhite">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="value" stdset="0">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberTime">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>118</red>
         <green>116</green>
         <blue>108</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="value" stdset="0">
    <double>20.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlack">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/prefix1/image/black.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelWhite">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/prefix1/image/white.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MyButton" name="buttonYes" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="MyButton" name="buttonNo" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="MyButton" name="buttonGoBackWin" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="MyButton" name="buttonGoon" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QWidget</extends>
   <header location="global">mybutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
